src/index.js

```diff
@@ -1 +1,275 @@
-import './index.css';
\ No newline at end of file
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+
+import './index.css';
+
+// Define the constants
+const mealList = document.getElementById('meal');
+const mealDetailsContent = document.querySelector('.meal-details-content');
+const recipeCloseBtn = document.getElementById('.recipe-close-btn');
+const url = 'https://www.themealdb.com/api/json/v1/1/filter.php?i=egg';
```

Comment:
- The web app uses an API that bases its development around it ✔
- The API should allow for getting a list of items with a unique item id and detailed information about the items. ✔
 
Well done 🙌🔥